packages/lisk-transactions/src/4_register_multisignature_account.js

```diff
@@ -17,11 +17,28 @@ import {
 	prependPlusToPublicKeys,
 	validateKeysgroup,
 	wrapTransactionCreator,
+	isValidInteger,
 } from './utils';
 
-const registerMultisignatureAccount = ({ keysgroup, lifetime, minimum }) => {
+const validateInputs = ({ keysgroup, lifetime, minimum }) => {
 	validateKeysgroup(keysgroup);
 
+	if (!isValidInteger(lifetime) || lifetime < 1 || lifetime > 72) {
```

Comment:
It would be better to split this magic numbers in to some place.
In core, these are network `constants`